mautic/mautic
# More autowiring

| Q                                      | A
| -------------------------------------- | ---
| New feature/enhancement? (use the a.x branch)      | [ x ]
| Automated tests included?              | [ x ] 



#### Description:
Continue with autowiring of controllers. Remove `CoreSubscriber` and inject services via `__construct`. Remove `initialize` method.



#### Steps to test this PR:
Probably best is to code review.

1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Check out export works, other parts of the system works.
